src/NCPkgPopupDiskspace.cc

```diff
@@ -232,7 +232,7 @@ void NCPkgDiskspace::checkRemainingDiskSpace( const ZyppPartitionDu & partition
     if ( totalSize != 0 )
 	percent = ( 100 * usedSize ) / totalSize;
 
-    int	free	= ( totalSize - usedSize ) / FSize::MB;
+    long long free = ( totalSize - usedSize ) / FSize::MB;
```

Comment:
Would it be possible to use `FSize` type directly? Both `totalSize` and `usedSize` have that type already.

Reply:
Oh, it's compared to `MIN_FREE_MB_PROXIMITY` which is in MB, so it must be converted to MB first...

So it's OK.

Reply:
Sure, but free has MiB as base while FSize always B. So simply using FSize would be confusing. Using FSize would also need all the checks later to be changed.